tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/PrimitiveInjectionTest.java

```diff
@@ -282,4 +282,21 @@ public void verifyInjectedCustomString() throws Exception {
         Reporter.log(reply.toString());
         Assert.assertTrue(reply.getBoolean("pass"), reply.getString("msg"));
     }
+
+    @RunAsClient
+    @Test(groups = TEST_GROUP_CDI_PROVIDER,
+        description = "Verify that the injected customString claim is as expected")
```

Comment:
Description is wrong as it is about testing Boolean and not String.

`String uri = baseURL.toExternalForm() + "endp/verifyInjectedCustomBoolean";`